code/modules/item_worth/worths_list.dm

```diff
@@ -493,6 +493,7 @@ var/list/worths = list(
 					/obj/item/clothing/suit/armor/centcomm = 2000,
 					/obj/item/clothing/suit/armor/heavy = 1000,
 					/obj/item/clothing/suit/armor/hos = 1000,
+					/obj/item/clothing/suit/storage/toggle/armor/hos/jensen = 1000,
```

Comment:
Pretty sure that this is needed, as its already covered by the HoS Armor price in the line above.

Please verify with the price scanner.

Reply:
Nope, 
`/obj/item/clothing/suit/storage/toggle/armor/hos` is not a child of 
`/obj/item/clothing/suit/armor/hos`.

Reply:
I think Lohikar is right - since my modifications makes `/obj/item/clothing/suit/storage/toggle/armor/hos` its own kind, that is not child of `/obj/item/clothing/suit/armor/hos`

Reply:
Right, oversaw that